lib/trento_web/plugs/app_jwt_auth_plug.ex

```diff
@@ -25,12 +25,21 @@ defmodule TrentoWeb.Plugs.AppJWTAuthPlug do
   def fetch(conn, _config) do
     with {:ok, jwt_token} <- read_token(conn),
          {:ok, claims} <- validate_access_token(jwt_token) do
+      sub = claims["sub"]
```

Comment:
Could we destructure the claims on line 27, also the sub one, it's safer to do since we need to fail asap if they don't match with the with clause

Reply:
Done https://github.com/trento-project/web/pull/3240/commits/b6a9b12e52381933e62e6a672b9156cb78afe79b